### PR TITLE
Jb14363

### PR DESCRIPTION
**Pull request by jpetrell**

### CODE REVIEW

**Comment by pvuorela** [comment]
Should also remove the files that are no longer used?

Generally LGTM

File: qml/plugins/CleanBackup.qml [hunk 1]
```diff
@@ -3,8 +3,10 @@ import Sailfish.Silica 1.0
 import Sailfish.Utilities 1.0
 
 ActionItem {
-    //% "Clean backup storage"
-    actionName: qsTrId("sailfish-tools-clean-backup")
+    //% "Backup"
+    title: qsTrId("sailfish-tools-he-backup")
+    //% "Clean"
+    actionName: qsTrId("sailfish-tools-bt-clean")
```
Comment (jpetrell) [inline qml/plugins/CleanBackup.qml]: Added comment to clarify purpose of the "Backup" title.

**Comment by jpetrell** [comment]
Thanks, removed files and code related to keyboard and device restarting.

> could consider defining ids that are used many times in one place.

I am used to defining multiple identical translation IDs if the same translation is needed in multiple places around the project. While duplication is bad coupling multiple places to a shared definition file also feels bad for such a simple case.

**Comment by pvuorela** [comment]
LGTM.

**Pull request merged**